linkis-commons/linkis-rpc/src/main/scala/org/apache/linkis/rpc/transform/RPCConsumer.scala

```diff
@@ -43,6 +46,7 @@ private[linkis] object RPCConsumer {
           if (data.isEmpty) return BoxedUnit.UNIT
           val objectStr = data.get(OBJECT_VALUE).toString
           val objectClass = data.get(CLASS_VALUE).toString
+          logger.debug(s"The corresponding anti-sequence is class $objectClass")
```

Comment:
should to add if (logger.isDebugEnabled())

Reply:
i will change the string format to  string template,  to avoid extra operator